src/parser/types.ts

```diff
@@ -507,7 +586,50 @@ interface DeterministicInterruption {
   components: (string)[]
 }
 
-interface RandomInterruption {
+/** The RandomInterruption interface is used to define an interruption that will be shown randomly in the block.
+ *
+ * For example, if you want to show a single interruption randomly in the block, you would set spacing to random and numInterruptions to 1. If you want to show 3 interruptions randomly in the block, you would set spacing to random and numInterruptions to 3.
+ *
+ * The components property is an array of the components that will be inserted randomly in the block. These components should reference components in the StudyConfig.components section of the config.
+ *
+ * Here's an example of how to use the RandomInterruption:
+ *
+ * ```js
+ * {
+ *   "order": "fixed",
+ *   "components": [
+ *     "component1",
+ *     "component2",
+ *     "component3",
+ *     "component4",
+ *     "component5",
+ *     "component6"
+ *   ],
+ *   "interruptions": [
+ *     {
+ *       "spacing": "random",
+ *       "numInterruptions": 3,
+ *       "components": [
+ *         "interruption1",
+ *         "interruption2"
+ *       ]
+ *     }
+ *   ]
+ * }
+ * ```
+ *
+ * The resulting sequence array could be:
+ *
+ * ```js
+ * [
+ *   ["component1", "interruption1", "interruption2", "component2", "component3", "component4", "component5", "component6"],
+ *   ["component1", "component2", "component3", "interruption1", "interruption2", "component4", "component5", "component6"],
+ *   ["component1", "component2", "component3", "component4", "component5", "interruption1", "interruption2", "component6"],
+ *   ...
+ * ]
+ * ```
+*/
```

Comment:
So you state that the number of interruptions is 3, but I only see two interruptions in the sequences. Also, will these interruptions always appear sequentially?

Reply:
Oh yeah, this is completely wrong... I'll fix it. They will always be sequential as defined above

Reply:
Fixed in [538d8f4](https://github.com/revisit-studies/study/pull/383/commits/538d8f4ff59e84058be214a3032637bb262018eb)